tests/agent_features/test_collector_payloads.py

```diff
@@ -63,3 +65,8 @@ def test_transaction_event_json():
 @validate_custom_event_collector_json()
 def test_custom_event_json():
     custom_event_application.get('/')
+
+@validate_log_event_collector_json
```

Comment:
```suggestion
@pytest.mark.xfail("Unwritten validator")
@validate_log_event_collector_json
```

Let's x-fail this and do a final testing cleanup PR to address this test, the validators, and any other issues.